test/Room-test.js

```diff
@@ -0,0 +1,78 @@
+import { expect } from 'chai';
+import Room from '../src/classes/Room';
+import sampleRooms from'../src/data/sampleRooms';
+
+const getRandomIndex = (array) => {
+  return array[Math.floor(Math.random() * array.length)];
+};
+
+describe('Room', () => {
+  let room;
+
+  beforeEach(() => {
+
+    room = new Room(getRandomIndex(sampleRooms))
+  });
+
+  it('should be a function', () => {
+    expect(Room).to.be.a('function');
+  });
+
+  it('should be an instance of Room', () => {
+    expect(room).be.be.an.instanceof(Room);
+  });
+
+  it('should have a room number that is a number', () => {
+    expect(room.number).to.be.a('number');
+  });
+
+  it('should have a room number that is a positive number', () => {
+    expect(room.number).to.be.greaterThan(0);
+  });
+
+  it('should have a room type that is a string', () => {
+    expect(room.roomType).to.be.a('string');
+  });
+
+//   it('should have a room type that is a string of residential suite, suite, junior suite, or single room', () => {
```

Comment:
Very curious about this. Hoping to find a slot today or Monday to ask an instructor.